<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Datos del Proyecto</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bAtras">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>390</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Atras</string>
    </property>
   </widget>
   <widget class="QLabel" name="lNombreSocio">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre del Socio</string>
    </property>
   </widget>
   <widget class="QLabel" name="lDescripcion">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Descripcion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eImporte">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eNombreSocio">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="eDescripcion">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>401</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lImporte">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Importe</string>
    </property>
   </widget>
   <widget class="QLabel" name="lFechaAbono">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha de abono</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eFechaAbono">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lModoPago">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Modo de pago</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eModoPago">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eNombreJoven">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lNombreJoven">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Joven Apadrinado</string>
    </property>
   </widget>
   <widget class="QLabel" name="lNumeroDeMovimiento">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>NÃºmero de movimiento</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eNumeroDeMovimiento">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
